src/app/styles/modules/_colors.scss

```diff
@@ -24,3 +24,5 @@ $activity-incomplete: #BEBEBE;
 $malaria-outer-circle-color: #E13946;
```

Comment:
do not do it this way. The idea to have this file is to have consistency in colors used in the application. Define it like this 
$maroon : #E13946 and then use it where ever you want. That way we will have just one shade of marron through out the app.

Reply:
@lunayach This comment here.. pLease follow this.